### PR TITLE
building html body

### PR DESCRIPTION
**Pull request by denmanma**

### CODE REVIEW

**Comment by jonathan-irvin** [comment]
Copy and paste your html into here: https://validator.w3.org/#validate_by_input and start by correcting the validation errors.

Usually, when you're doing stuff with React/Angular, the html won't always be valid, but since we're starting with the basics, you learn a lot by checking for valid HTML/CSS.

File: resume.html [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+	<title>
+	Resume: MICHAEL DENMAN
+	</title>
+	</head>
+
+	<body>
+		<h2>
+		<img id="Self Photo" src="self-photo.png" width="80" height="105"> MICHAEL ALEXANDER DENMAN
+		</h2>
+	325-277-2340 - denmanmd@gmail.com <a href='https://www.linkedin.com/in/michael-denman/'> LinkedIn Profile</a></br>
```
Comment (jonathan-irvin) [inline resume.html]: We can add hyperlinks for both your email and phone.

Comment (jonathan-irvin) [reply resume.html]: For phones, prefix the link href with `tel:` and emails with `mailto:`
Try clicking on them after those changes.

File: resume.html [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+inspection programs tracking over 47K inspections. Drafted maintenance guide distributed
+to Air Force database admins and users across the globe.</br>
+- Sole QA inspector for more than 40 personnel and 2000 avionics components. Ensured
+safety and legal compliance with local, national, and international law, earning the Air Force
+Commendation Medal.</br>
+- Maintained high attention to detail. Identified and documented over 200 safety and
+compliance discrepancies and provided corrective courses of action to responsible
+supervisors to streamline processes and prevent future occurrences.</br>
+
+<h3>OCT 2005 - MONTH 2012</h3>
```
Comment (jonathan-irvin) [inline resume.html]: MONTH

File: resume.html [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+Avionics Test Station & Components Technician - USAF, Abilene, TX & Okinawa, Japan</br>
+Performed and managed avionics test station functions and activities. Operated, inspected,
+maintained, programmed, and calibrated computer and manually operated avionics test equipment,
+support equipment, and aircraft avionics systems components.</br>
+- Skilled troubleshooter responsible for repair and return to service of over 1000 aircraft
+parts. Saved over $500M in replacement costs through a 7-year period, earning an Air
+Force Achievement Medal.</br>
+- Lead technician responsible for software updates and system upgrades for 13
+Windows-based test stations leading to over 99% up time of required machines.</br>
```
Comment (jonathan-irvin) [inline resume.html]: Is it up time or up-time?

File: resume.html [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+<h2>Certificates</h2>
+- (2018) CompTIA Advanced Security Practitioner (CASP+), San Antonio, TX</br>
+- (2017) Computer Network Operations(CNO) Exploitation Analyst, San Antonio, TX</br>
+- (2016) Joint Cyber Analysis Course (Intro to IT), Pensacola, FL (Honor Graduate)</br>
+- (2014) Digital Network Intelligence Analysis Course, San Angelo, TX (Honor Graduate)</br>
+
+</br></br></br>
+
+Case number (RES-2019-01941)
+
```
Comment (jonathan-irvin) [inline resume.html:165]: </body>

**Review by jonathan-irvin — CHANGES_REQUESTED** [review]
Other than some validation errors which will overlap with some of my comments, you're off to a good start!

File: resume.html (hunk 1 above)
Comment (jonathan-irvin) [reply resume.html]: You'll be replacing https or http, so a phone would look like `<a href="tel:+13252772340">325-277-2340</a>`

**Comment by jonathan-irvin** [comment]
@denmanma let's work on correcting these errors and getting this PR merged.

File: resume.html (hunk 3 above)
Comment (denmanma) [reply resume.html]: up-time. good catch. I've had several people look over these bullets, and nobody has caught that.

File: resume.html (hunk 4 above)
Comment (denmanma) [reply resume.html:165]: What?

Comment (denmanma) [reply resume.html:165]: Turns out "\</br\>" is not acceptable. :P

**Pull request merged**